common/src/main/proto/TransportMessages.proto

```diff
@@ -551,3 +554,19 @@ message PbReadAddCredit {
   int64 streamId = 1;
   int32 credit = 2;
 }
+
+message PbStreamChunkSlice {
+  int64 streamId = 1;
+  int32 chunkIndex = 2;
+  int32 offset = 3;
+  int32 len = 4;
+}
+
+message PbChunkFetchRequest {
+  PbStreamChunkSlice streamChunkSlice = 1;
+}
+
+message PbTransportableError {
+  int64 streamId = 1;
+  string errorMessage = 2;
```

Comment:
LGTM, exception a nit: errorMessage -> message (message of TransportableError has the `error` meaning).